.github/workflows/run_tests.yml

```diff
@@ -25,4 +25,4 @@ jobs:
           composer install --prefer-dist --no-interaction
 
       - name: Execute tests
-        run: ./vendor/bin/phpunit
+        run: ./vendor/bin/phpunit tests
```

Comment:
Is the config file being used? Because the config file already states that the `tests/` directory is part of out test suite.